hack/make-bundle-vars.mk

```diff
@@ -46,7 +46,9 @@ CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
 
 # A comma-separated list of bundle images (e.g. make catalog-build BUNDLE_IMGS=example.com/operator-bundle:v0.1.0,example.com/operator-bundle:v0.2.0).
 # These images MUST exist in a registry and be pull-able.
-BUNDLE_IMGS ?= $(BUNDLE_IMG)
+OCS_BUNDLE_IMG ?= quay.io/ocs-dev/ocs-operator-bundle:latest
+IBM_BUNDLE_IMG ?= docker.io/ibmcom/ibm-storage-odf-operator-bundle:0.2.0
+BUNDLE_IMGS ?= $(BUNDLE_IMG),$(OCS_BUNDLE_IMG),$(IBM_BUNDLE_IMG)
```

Comment:
Can we do something to optionally concatenate these strings? It's possible we may want to sometimes generate the catalog index with only odf-operator bundles.

Reply:
What should be the default behavior?

Reply:
We should also be more careful on which tags of the bundle images we want (e.g. something other than `latest` for ocs-operator without having to write the entire URL), but that can also be a later PR.

Reply:
1, 2, 3 are done